### PR TITLE
[BHX] Add new organisers and a sponsor

### PR DESCRIPTION
**Pull request by Waterdrips**
I have emailed info@devopsdays.org to add Dan and Maria - not sure if its been done yet.

### CODE REVIEW

File: data/events/2023-birmingham-uk.yml [hunk 1]
```diff
@@ -68,6 +68,13 @@ team_members: # Name is the only required field for team members.
     employer: "Kasten by Veeam"
     image: "michael-cade.jpg"
     twitter: "@MichaelCade1"
+  - name: "Dan Webb"
+    twitter: "dan_webb"
+    github: "damacus"
```
Comment (damacus) [inline data/events/2023-birmingham-uk.yml:73]:
```suggestion
  - name: "Dan Webb"
    employer: "Independent Consultant"
    twitter: "dan_webb"
    github: "damacus"
```

**Review by damacus — APPROVED** [review]

File: data/events/2023-birmingham-uk.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ enddate:  2023-06-16T09:00:00+00:00 # The end date of your event. Leave blank if
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
 cfp_date_start:  2022-11-01T09:00:00+00:00 # start accepting talk proposals.
-cfp_date_end:  2023-04-30T09:00:00+00:00 # close your call for proposals.
+cfp_date_end:  2023-03-30T09:00:00+00:00 # close your call for proposals.
```
Comment (phrawzty) [inline data/events/2023-birmingham-uk.yml:21]: This is a fairly significant change that isn't addressed in the title or comment of the PR. Are you sure about this?

**Comment by phrawzty** [comment]
I've made the changes to the mailing list. Once the review comment is addressed, the rest is fine. :)

**Comment by jyee** [comment]
The comment around `cfp_date_end` was covered in Slack (CFP has indeed closed already). I'm merging this.

**Pull request merged**